app/views/pipeline_outputs/show.html.erb

```diff
@@ -40,27 +40,20 @@
 <table class="table table-condensed">
       <thead>
         <tr>
-          <th>Taxon</th>
-          <th>Taxon Level</th>
-          <th>Count Type</th>
-          <th>Count</th>
+          <th>Species</th>
+          <th>NT Species Count</th>
+          <th>NR Species Count</th>
         </tr>
       </thead>
       <tbody>
-        <% @pipeline_output.taxon_counts.sort_by(&:count).reverse.each do |element| %>
+        <% counts_display = @pipeline_output.taxon_counts %>
```

Comment:
you might be able to clean this up by implementing it as a ActiveRecord scope: http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Named/ClassMethods.html